.github/workflows/macos_build_ios.yml

```diff
@@ -14,171 +14,171 @@ jobs:
-
-    - name: Slack notification
-      env:
-        SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
-      run: |
-        payload="{\"channel\": \"#notifications-git\", \"username\": \"webhookbot\", \"text\": \"Finished GitHub action $GITHUB_WORKFLOW for $GITHUB_REF (TreeMapper)\"}"
-        curl -X POST --data-urlencode "payload=$payload" https://hooks.slack.com/services/$SLACK_WEBHOOK_URL
+      - name: Checkout branch
+        uses: actions/checkout@v2
+
+      - name: Select Xcode
+        run: |
+          echo "Default version:" `xcode-select -p`
+          sudo xcode-select -s /Applications/Xcode_13.1.app
+          echo "Selected version:" `xcode-select -p`
+
+      - name: Upgrade npm to latest version
+        run: sudo npm i -g npm@latest --force
+
+      - name: Check versions
+        run: |
+          echo "Xcode version:"
+          xcodebuild -version
+          echo "Pod version:"
+          pod --version
+          echo "Brew version:"
+          brew --version
+          echo "Node version:"
+          node --version
+          echo "NPM version:"
+          npm --version
+          echo "PATH:"
+          echo $PATH
+
+      - name: Installing and changing to newer bash version
+        run: |
+          echo $PATH
+          bash -version
+          brew reinstall bash
+          sudo chsh -s /usr/local/bin/bash
+
+      - name: Installing and changing to gnu sed version
+        run: |
+          echo $PATH
+          brew reinstall gnu-sed
+          echo 'export PATH="/usr/local/opt/gnu-sed/libexec/gnubin:$PATH"' >> ~/.bashrc
+
+      - name: Installing google-drive-upload script
+        env:
+          CLIENT_ID: ${{ secrets.GOOGLE_DRIVE_CLIENT_ID }}
+          CLIENT_SECRET: ${{ secrets.GOOGLE_DRIVE_CLIENT_SECRET }}
+          REFRESH_TOKEN: ${{ secrets.GOOGLE_DRIVE_REFRESH_TOKEN }}
+          ACCESS_TOKEN: ${{ secrets.GOOGLE_DRIVE_ACCESS_TOKEN }}
+        run: |
+          source ~/.bashrc
+          echo $PATH
+          curl --compressed -s https://raw.githubusercontent.com/labbots/google-drive-upload/v2.7/install.sh | bash -s -- -R v2.7
+          echo "CLIENT_ID=\"$CLIENT_ID\"" >> ~/.googledrive.conf
+          echo "CLIENT_SECRET=\"$CLIENT_SECRET\"" >> ~/.googledrive.conf
+          echo "REFRESH_TOKEN=\"$REFRESH_TOKEN\"" >> ~/.googledrive.conf
+          echo "ACCESS_TOKEN=\"$ACCESS_TOKEN\"" >> ~/.googledrive.conf
+          echo "ACCESS_TOKEN_EXPIRY=" >> ~/.googledrive.conf
+          echo "ROOT_FOLDER=\"1sO16rDuJA8McRA8-Q-I6R2OjMGcG4nEp\"" >> ~/.googledrive.conf
+          echo "ROOT_FOLDER_NAME=" >> ~/.googledrive.conf
+
+      - name: Caching node modules
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-treemapper2-${{ hashFiles('package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-treemapper2-
+
+      - name: Install npm
+        run: |
+          npm install --legacy-peer-deps
```

Comment:
Please merge develop where I already added `--legacy-peer-deps` and omit all your reformatting changes of all the GitHub workflow files.

Reply:
We can also keep your version.